mmdet/datasets/voc.py

```diff
@@ -59,24 +59,28 @@ def evaluate(self,
             raise KeyError(f'metric {metric} is not supported')
         annotations = [self.get_ann_info(i) for i in range(len(self))]
         eval_results = OrderedDict()
+        iou_thrs = [iou_thr] if isinstance(iou_thr, float) else iou_thr
         if metric == 'mAP':
-            assert isinstance(iou_thr, float)
+            assert isinstance(iou_thrs, list)
             if self.year == 2007:
                 ds_name = 'voc07'
             else:
                 ds_name = self.CLASSES
-            mean_ap, _ = eval_map(
-                results,
-                annotations,
-                scale_ranges=None,
-                iou_thr=iou_thr,
-                dataset=ds_name,
-                logger=logger)
-            eval_results['mAP'] = mean_ap
+            mean_aps = []
+            for iou_thr in iou_thrs:
```

Comment:
It seems to be a little bit weird that we put the logic of processing multiple iou_thr outside the eavl_map function. This forces the users to write similar logic every time we call eval_map. Should we put the logic into eval_map?